### DISCUSSION TITLE
建好的API 不能删除

### DISCUSSION THREAD

**Discussion by markla2014** [Postcatlab/postcat#189] [General]
请问API 删除键在哪里？

**Comment by scarqin** [comment]
@markla2014 麻烦截图下你的界面，你是内网/离线使用吗？

**Comment by markla2014** [comment]
内网离线安装

![无标题](https://user-images.githubusercontent.com/11838376/202389467-77c6b0e4-e943-4f6c-bafa-f75f155df5bf.png)

**Comment by scarqin** [comment]
@markla2014 了解了，是 icon 图标用了在线资源导致的问题，已经修复了，本周将会发布。
可以 hover 相应位置删除。
<img width="405" alt="image" src="https://user-images.githubusercontent.com/15225835/202397146-0374681b-499c-4849-addd-68204a29b8bf.png">

**Comment by markla2014** [comment]
好的，谢谢

**Reply by markla2014** [reply]
试了一下，右键在无线情况也失效了，期望本期发版可以修复

**Reply by scarqin** [reply]
@markla2014 右键操作什么？

**Reply by scarqin** [reply]
@markla2014 我们暂不支持右键操作，可以提 issue 建议一下～

**Reply by markla2014** [reply]
没啥，hover 也失效了
